framework/assets/yii.validation.js

```diff
@@ -236,6 +236,23 @@ yii.validation = (function ($) {
             }
         },
 
+        _trim: function(value, options = {skipOnEmpty: true}) {
```

Comment:
I'd move it out of public scope instead of exposing it with `_` prefix.

Reply:
@samdark `_trim` - polyfill, `trim` - validator. you can use polyfill in other places

Reply:
No you can't. If it's prefixed, it's meant to be private so it's better to move it out of scope.

Reply:
@samdark done